arcsi/view/show.py

```diff
@@ -30,7 +35,8 @@ def view_show(id):
     show = archon_view_show(id)
     if hasattr(show, "status_code") and show.status_code == 404:
         return "Show not found"
-    return render_template("show/view.html", show=show)
+    empty_playlist = show_playlist_is_empty(show.playlist_name)
+    return render_template("show/view.html", show=show, empty_playlist=empty_playlist)
```

Comment:
Great!!

Reply:
Here I missed that part when the playlist is not even exists, now that's also covered :)